platform/commonUI/general/test/controllers/DateTimePickerControllerSpec.js

```diff
@@ -57,6 +65,119 @@ define(
                 );
             });
 
+            it("updates value in model when values in scope change", function () {
+                mockScope.date = {
+                    year: 1998,
+                    month: 0,
+                    day: 6
+                };
+                mockScope.time = {
+                    hours: 12,
+                    minutes: 34,
+                    seconds: 56
+                };
+                fireWatchCollection("date", mockScope.date);
+                expect(mockScope.ngModel[mockScope.field])
+                    .toEqual(moment.utc("1998-01-06 12:34:56").valueOf());
+            });
+
+            describe("once initialized with model state", function () {
+                var testTime = moment.utc("1998-01-06 12:34:56").valueOf();
+
+                beforeEach(function () {
+                    fireWatch("ngModel[field]", testTime);
+                });
+
+                it("exposes date/time values in scope", function () {
+                    expect(mockScope.date.year).toEqual(1998);
+                    expect(mockScope.date.month).toEqual(0); // Months are zero-indexed
+                    expect(mockScope.date.day).toEqual(6);
+                    expect(mockScope.time.hours).toEqual(12);
+                    expect(mockScope.time.minutes).toEqual(34);
+                    expect(mockScope.time.seconds).toEqual(56);
+                });
+
+                it("provides names for time properties", function () {
+                    Object.keys(mockScope.time).forEach(function (key) {
+                        expect(mockScope.nameFor(key))
+                            .toEqual(jasmine.any(String));
+                    });
+                });
+
+                it("provides options for time properties", function () {
+                    Object.keys(mockScope.time).forEach(function (key) {
+                        expect(mockScope.optionsFor(key))
+                            .toEqual(jasmine.any(Array));
+                    });
+                });
+
+                it("exposes times to populate calendar as a table", function () {
```

Comment:
I don't quite follow how this test description relates to the test implementation. The test seems to be searching the table for a particular cell? A comment or rephrasing of the description might clarify the purpose of this test